infrastructure/kctf-samples/apache-php/challenge/config/nsjail-php.cfg

```diff
@@ -28,6 +28,17 @@ mount: [
     dst: "/tmp"
     fstype: "tmpfs"
     rw: true
+  },
+  {
+    dst: "/mnt"
+    fstype: "tmpfs"
+    rw: true
+  },
+  {
+    src: "/mnt/disks/gcs-task"
```

Comment:
Do you need to mount the whole gcs-task directory? Can't you mount the subdirectories instead?
E.g. I'm worried that when you touch it from the outside (mkdir -p /mnt/disks/gcs-task/sessions), someone might have replaced sessions with a symlink.

Reply:
for this specific usecase, we could map ./sessions to /var/lib/php/sessions (instead of making it a symlink in the dockerfile) and ./uploads to /usr/var/uploads or something like that.

however, I don't think users will know or understand this is a security risk in other contexts, let's say we, outside the chroot, as root do something like:
```bash
cat /mnt/disks/gcs-task/foo > /mnt/disks/gcs-task/bar
```

Then I imagine that /mnt/disks/gcs-task/bar could be a symlink pointing to somewhere dangerous (say, /root/.ssh-keys or similar), could we protect this somehow?

Reply:
we now map subdirs instead